pkg/mintycfg/runner.go

```diff
@@ -0,0 +1,105 @@
+// Copyright 2024 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// The mintycfg package contains a CLI command that can be used to validate minty
+// configuration files.
+package mintycfg
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/fs"
+	"os"
+
+	"github.com/google/cel-go/cel"
+
+	"github.com/abcxyz/github-token-minter/pkg/config"
+)
+
+type singleFileConfigLoader struct {
+	filePath string
+}
+
+// Load reads the contents of configuration files from the local file system.
+func (l *singleFileConfigLoader) Load(ctx context.Context, org, repo string) (*config.Config, error) {
+	data, err := os.ReadFile(l.filePath)
+	if err != nil {
+		// file not found is not an error
```

Comment:
why? shouldnt this throw an error? what happens if i give a file that doesnt exist?

Reply:
Eh, this was a copy from the main local config loader and a file not found there isn't an error. I'll remove it from here.